tests/Integration/SettingsTest.php

```diff
@@ -40,41 +40,6 @@ public function setUp(): void
         $this->application = new Application(self::$kernel);
     }
 
-    public function testGetDefaultSettings(): void
```

Comment:
I really think we should not be testing the `meilisearch-php` in a client (in this case Symfony lib). This test belongs to the meilisearch-php :)